pkg/project/project.go

```diff
@@ -608,14 +610,23 @@ func (e *Event) Name() string { return e.event.Name }
 
 // Value returned with all secrets evaluated
 func (e *Event) Value() (string, error) {
+	if e.BashifyPaths {
+		return ExpandFromProjectBashifyPaths(e.event.Value, e.project)
+	}
 	return ExpandFromProject(e.event.Value, e.project)
 }
 
 // Scope returns the scope property of the event
 func (e *Event) Scope() ([]string, error) {
 	result := []string{}
 	for _, s := range e.event.Scope {
-		v, err := ExpandFromProject(s, e.project)
+		var v string
+		var err error
+		if e.BashifyPaths {
+			v, err = ExpandFromProjectBashifyPaths(s, e.project)
```

Comment:
Like Value; this would lead to us bashifying values that aren't paths. I don't think we need to be expanding this value for bash paths for bash compatibility to work?

Reply:
Like previously, this only instructs how to handle a `.path()` that was encountered.